app/squac/settings.py

```diff
@@ -40,6 +40,7 @@
 INTERNAL_IPS = [
     'localhost',
     '10.0.2.2',
+    '128.95.16.34'
```

Comment:
was this for testing, should it be removed?

Reply:
Yes, I removed it